frontend/src/__tests__/cypress/cypress/e2e/pipelines/Pipelines.cy.ts

```diff
@@ -116,9 +116,155 @@ describe('Pipelines', () => {
     cy.wait('@refreshVersions');
 
     // Verify the uploaded pipeline version is in the table
-    pipelinesTable.toggleExpandRowByIndex(0);
+    pipelinesTable.toggleExpandRowByName(initialMockPipeline.name);
     pipelinesTable.findRowByName('New pipeline version');
   });
+
+  it('delete single pipeline version', () => {
+    initIntercepts();
+    initPipelineIntercepts();
+    pipelinesGlobal.visit(projectName);
+    let deleted = false;
+    cy.intercept(
+      {
+        pathname: `/api/proxy/apis/v1beta1/pipeline_versions/${initialMockPipelineVersion.id}`,
+        method: 'POST',
+      },
+      (req) => {
+        deleted = true;
+        req.reply({ body: {} });
```

Comment:
Yes, if we don't return the body, the alert will be fired. I think it's because of the structure of how we capture KubeFlow error.

![Screenshot 2024-02-01 at 11 50 45 AM](https://github.com/opendatahub-io/odh-dashboard/assets/37624318/c057e3b7-5836-4b9b-b6d4-8566155ea88c)